modules/mono/editor/GodotTools/GodotTools/Internals/GodotSharpDirs.cs

```diff
@@ -48,12 +48,6 @@ public static string DataEditorToolsDir
             }
         }
 
-        public static void RegisterProjectSettings()
-        {
-            GlobalDef("dotnet/project/assembly_name", "");
-            GlobalDef("dotnet/project/solution_directory", "");
-        }
```

Comment:
I still don't understand why this wasn't working. It should be called at the start of the editor.

Reply:
`GDMono::_load_project_assembly` tries to get `dotnet/project/assembly_name`. AFAIK, in order to enable the plugin, which is implemented in C#, we need to load the assembly so that gets called first.